openmc/source.py

```diff
@@ -123,17 +224,21 @@ def __init__(
                 self.domain_type = 'universe'
             self.domain_ids = [d.id for d in domains]
 
+    @property
+    def type(self) -> str:
+        return 'independent'
```

Comment:
Need to confirm whether this should be `"independent"` or `"independent_source"`

Reply:
I'm going to opt for `"independent"` to be consistent with what the other source classes return. The `_source` seems redundant to me due to the context of the in the two places it's used: as an attribute of a source class in the Python API and an attribute for XML source nodes in our XML files.